pages/nft/iscn/_iscnId.vue

```diff
@@ -204,6 +224,40 @@ export default class NFTTestMintPage extends Vue {
     return formData
   }
 
+  get detailsPageURL(): string {
+    return `${LIKER_LAND_URL}/nft/class/${this.classId}`
```

Comment:
Not sure if `classId` can contains anything that requires `encodeURIComponent`, safer to call it?